config/database.yml

```diff
@@ -0,0 +1,15 @@
+development:
+  adapter: postgresql
+  encoding: unicode
+  database: blog_app_development
+  pool: 5
+  username: blog_app
+  password: blog_app
```

Comment:
- [ ] [OPTIONAL] I also suggest the use of environment variables to store sensitive data. For example you can make use of the rails inbuilt credentials - `<%= Rails.application.credentials.username %>` then store the values in the encrypted credentials file. To open it use `rails credentials:edit` in the terminal

Reply:
thanks done